bundles/plugins/org.bonitasoft.studio.common.repository/src/org/bonitasoft/studio/common/repository/Repository.java

```diff
@@ -161,16 +163,11 @@ public Repository create(final IProgressMonitor monitor) {
     }
 
     protected CreateBonitaBPMProjectOperation newProjectWorkspaceOperation(final String projectName, final IWorkspace workspace) {
-        return new CreateBonitaBPMProjectOperation(workspace, projectName).
-                addNature("org.eclipse.xtext.ui.shared.xtextNature").
-                addNature("org.bonitasoft.studio.common.repository.bonitaNature").
-                addNature(JavaCore.NATURE_ID).
-                addNature("org.eclipse.pde.PluginNature").
-                addNature("org.eclipse.jdt.groovy.core.groovyNature").
-                addBuilder("org.eclipse.jdt.core.javabuilder").
-                addBuilder("org.eclipse.xtext.ui.shared.xtextBuilder").
-                addBuilder("org.eclipse.pde.ManifestBuilder").
-                addBuilder("org.eclipse.pde.SchemaBuilder");
+        return new CreateBonitaBPMProjectOperation(workspace, projectName).addNature("org.eclipse.xtext.ui.shared.xtextNature")
+                .addNature("org.bonitasoft.studio.common.repository.bonitaNature").addNature(JavaCore.NATURE_ID).addNature("org.eclipse.pde.PluginNature")
```

Comment:
previous formatting was more readable (and don't we have some constants for al these values?)